spec/features/clearance/visitor_resets_password_spec.rb

```diff
@@ -0,0 +1,65 @@
+require 'rails_helper'
+require 'support/features/clearance_helpers'
+
+RSpec.feature 'Visitor resets password' do
+  before { ActionMailer::Base.deliveries.clear }
+
+  around do |example|
+    original_adapter = ActiveJob::Base.queue_adapter
+    ActiveJob::Base.queue_adapter = :inline
+    example.run
+    ActiveJob::Base.queue_adapter = original_adapter
+  end
+
+  scenario 'by navigating to the page' do
+    visit sign_in_path
+
+    click_link_or_button I18n.t('sessions.form.forgot_password')
+    expect(have_current_path).to eq new_password_path
```

Comment:
Not sure if you manually changed this or rubocop broke it for you, but this is what it wanted per https://www.rubydoc.info/gems/rubocop-rspec/1.19.0/RuboCop/Cop/RSpec/Capybara/CurrentPathExpectation

```suggestion

    expect(page).to have_current_path(new_password_path)
```

Reply:
THANK YOU!